boards/arm/nucleo_f091rc/doc/nucleof091rc.rst

```diff
@@ -0,0 +1,168 @@
+.. _nucleo_f091rc_board:
+
+ST Nucleo F091RC
+################
+
+Overview
+********
+The STM32 Nucleo-64 development board with STM32F091RC MCU, supports Arduino and ST morpho connectivity.
+
+The STM32 Nucleo board provides an affordable, and flexible way for users to try out new concepts,
+and build prototypes with the STM32 microcontroller, choosing from the various
+combinations of performance, power consumption and features.
```

Comment:
add the comma before "and":

     power consumption, and features.

Reply:
Done